### PR TITLE
DEVDOCS-4761:[update] Add warning and update node version

### PR DESCRIPTION
**Pull request by bc-traciporter**
Changed node version to 18.15.0
Added warning about the deprecation of node-sass

Here is a summary of my changes.
1. Updated node version from 14.20.0 to 18.15.0 in the following files:

- apps-quick-start.mdx
- /pos/foundation-guide.mdx
- /subscriptions/foundation-guide.mdx
-  installing-stencil.mdx

2. Added a warning message to the following files:

-  installing-stencil.mdx
-  stencil-cli-options-and-commands.mdx
-  troubleshooting-your-setup.mdx

3. Added instructions on how to update your system in the following file:

-  troubleshooting-your-setup.mdx

### CODE REVIEW

File: docs/stencil-docs/installing-stencil-cli/installing-stencil.mdx [hunk 1]
```diff
@@ -1,11 +1,15 @@
 # Installing Stencil CLI
 
+<Callout type="warning">
+BigCommerce is currently targeting 11/1/2023 to sunset node-sass. To ensure that your storefront is up to date, use the latest node version (Node 18) in Stencil CLI and install the automated script to resolve [incompatible SCSS directives](/stencil-docs/installing-stencil-cli/troubleshooting-your-setup#incompatible-scss-directives), which can cause issues with the styling of your storefront.
```
Comment (zvuki) [inline docs/stencil-docs/installing-stencil-cli/installing-stencil.mdx:4]:
```suggestion
BigCommerce is currently targeting 11/1/2023 to sunset its node-sass fork in favor of the latest [sass/node-sass](https://github.com/sass/node-sass). To ensure that your storefront is up to date, use the latest node version (Node 18) in Stencil CLI and install the automated script to resolve [incompatible SCSS directives](/stencil-docs/installing-stencil-cli/troubleshooting-your-setup#incompatible-scss-directives), which can cause issues with the styling of your storefront.

```

File: docs/stencil-docs/installing-stencil-cli/stencil-cli-options-and-commands.mdx [hunk 2]
```diff
@@ -4,6 +4,10 @@
 
 This article is a comprehensive command reference for Stencil CLI, BigCommerce's powerful theme development and deployment tool. For installation instructions for your OS, see [Installing Stencil CLI](/stencil-docs/installing-stencil-cli/installing-stencil). For more information on BigCommerce's Stencil Theme Engine, see [About Stencil](/stencil-docs/getting-started/about-stencil). Continue reading below for detailed information on each Stencil CLI command and option.
 
+<Callout type="warning">
+BigCommerce is currently targeting 11/1/2023 to sunset node-sass. To ensure that your storefront is up to date, use the latest node version (Node 18) in Stencil CLI and install the automated script to resolve [incompatible SCSS directives](/stencil-docs/installing-stencil-cli/troubleshooting-your-setup#incompatible-scss-directives), which can cause issues with the styling of your storefront.
```
Comment (zvuki) [inline docs/stencil-docs/installing-stencil-cli/stencil-cli-options-and-commands.mdx:8]: could you use the same wording as above https://github.com/bigcommerce/dev-docs/pull/1855/files#diff-d653d450312b9b922608cb1e4495dbac19f5a10cc1d79e7d54c5572ab4187ddeR4

File: docs/stencil-docs/installing-stencil-cli/troubleshooting-your-setup.mdx [hunk 3]
```diff
@@ -6,6 +6,36 @@ For any unexpected behavior you encounter while developing your Stencil theme, w
 
 In some cases, the terminal will provide a verbose error message specifying where to look for problems. It has the potential to provide further insight on the issue. Diagnostic suggestions are listed on this page for error messages that may not be helpful in revealing the issue you're experiencing.
 
+<Callout type="warning">
+BigCommerce is currently targeting 11/1/2023 to sunset node-sass. To ensure that your storefront is up to date, use the latest version of node (Node 18) in Stencil CLI and install the automated script described in [Incompatible SCSS directives](#incompatible-scss-directives) to prevent an issue with the styling of your storefront.
```
Comment (zvuki) [inline docs/stencil-docs/installing-stencil-cli/troubleshooting-your-setup.mdx:10]: here as well

File: docs/stencil-docs/installing-stencil-cli/troubleshooting-your-setup.mdx [hunk 4]
```diff
@@ -6,6 +6,36 @@ For any unexpected behavior you encounter while developing your Stencil theme, w
...
+<Callout type="warning">
+BigCommerce is currently targeting 11/1/2023 to sunset its node-sass fork in favor of the latest [sass/node-sass](https://github.com/sass/node-sass). To ensure that your storefront is up to date, use the latest node version (Node 18) in Stencil CLI and use the CLI command to resolve [incompatible SCSS directives](/stencil-docs/installing-stencil-cli/troubleshooting-your-setup#incompatible-scss-directives), which can cause issues with the styling of your storefront.
+</Callout>
+
+## Incompatible SCSS directives
+
+To ensure that your storefront is up to date, perform the following steps:
+
+1. Using the latest version of Stencil CLI, run the stencil bundle command to validate your theme's code. Note any errors in the console output.
+```shell copy
```
Comment (bc-andreadao) [inline docs/stencil-docs/installing-stencil-cli/troubleshooting-your-setup.mdx:18]:
```suggestion
```shell showLineNumbers copy
```
The [code sample right below](https://github.com/bigcommerce/dev-docs/pull/1855/files#diff-980b386a9b8b6e6cd112b26ae164b46cd8241de9adf9712b0e8a294d6a6400a6R22) shows the line numbers even though it is only one line

File: docs/stencil-docs/installing-stencil-cli/troubleshooting-your-setup.mdx [hunk 5]
```diff
@@ -6,6 +6,36 @@ For any unexpected behavior you encounter while developing your Stencil theme, w
...
+stencil bundle
+```
+2. Update your files to ensure the correct formatting by running the following script. This step allows you to see changes before merging.
+```shell showLineNumbers copy
```
Comment (bc-andreadao) [inline docs/stencil-docs/installing-stencil-cli/troubleshooting-your-setup.mdx:22]: The other code samples on this page don't show line numbers if they are only one line, so we should be consistent across the entire page

**Review by bc-andreadao — APPROVED** [review]

File: docs/stencil-docs/installing-stencil-cli/installing-stencil.mdx [hunk 6]
```diff
@@ -111,7 +116,7 @@ If you're a pro at installing and configuring Python and Node.js environments on
 **Required Dependencies:**
 * [Git](https://git-scm.com/downloads) - required to run npm install
 * [Python 2.7.x](https://www.python.org/downloads/) - required to build some dependencies
-* [Node.js 14 and npm](https://nodejs.org/en/download/releases/) - later versions not currently supported on Windows
+* [Node.js 18.15.0 and npm](https://nodejs.org/en/download/releases/) - later versions not currently supported on Windows
```
[the remark this answers is not in the document]
Comment (jairo-bc) [reply docs/stencil-docs/installing-stencil-cli/installing-stencil.mdx:119]: Basically, any version of node 18 should be supported, cause there shouldn't be any breaking changes. 
But the latest version I've checked was 18.15

Comment (zvuki) [inline docs/stencil-docs/installing-stencil-cli/installing-stencil.mdx:119]:
```suggestion
* [Node.js 18.15.0 and npm](https://nodejs.org/en/download/releases/)
```
@jairo-bc I suppose this should be updated then ^

Comment (jairo-bc) [reply docs/stencil-docs/installing-stencil-cli/installing-stencil.mdx:119]: right, not sure from where did it come to

**Pull request merged**